x/wasm/ibctesting/coordinator.go

```diff
@@ -217,11 +217,9 @@ func (coord *Coordinator) ConnOpenInitOnBothChains(path *Path) error {
 		return err
 	}
 
-	if err := path.EndpointB.UpdateClient(); err != nil {
-		return err
-	}
+	err := path.EndpointB.UpdateClient()
```

Comment:
This and others: no need to keep an var:
```suggestion
	return path.EndpointB.UpdateClient()
```